### PR TITLE
Hotfix for Vietnamese flag

### PR DESCRIPTION
**Pull request by acapela000**
- Add `vi` - `vn` (Vietnam - South East Asia).
- Upgrade the Patch version.

### CODE REVIEW

File: src/components/select-language/select-language.tsx [hunk 1]
```diff
@@ -232,6 +232,7 @@ export const langToFlagIconMap: Record<string, string> = {
   "yo": "ng", // Yoruba (Nigeria)
   "zu": "za", // Zulu (South Africa)
   "xh": "za", // Xhosa (South Africa)
+  "vi": "vn", // Vietnam (South East Asia)
```
Comment (ixrock) [inline src/components/select-language/select-language.tsx]: Hello! 
Thanks for attempt to localise extension to Vietnamese, but you forgot most important pat: localization strings itself that must be in `_locales/vi/messages.json`.
Please copy-paste folder `_locales/en`, rename and update all messages there to your local language.

File: manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "XTranslate",
   "description": "__MSG_description__",
-  "version": "2.76.2",
+  "version": "2.76.3",
```
Comment (ixrock) [inline manifest.json]: Please rollback. There is no need to maintain new version for you _(it might be different when it goes to production)._

**Review by ixrock — CHANGES_REQUESTED** [review]

File: src/components/select-language/select-language.tsx (hunk 1 above)
Comment (acapela000) [reply src/components/select-language/select-language.tsx]: Hello!
Thanks for replying me. I saw that you already made the Fix and I also added the translation of Vietnamse. Let me know if you need me to change something else.

Comment (ixrock) [reply src/components/select-language/select-language.tsx]: All good now! Thanks for translation!

**Pull request merged**